packaging/desktop/package_darwin.sh

```diff
@@ -196,9 +197,16 @@ create_zip() {
   ditto -c -k --sequesterRsrc --keepParent $app_name.app $zip_name
 }
 
-s3sync() {
-  if [ ! "$bucket_name" = "" ] && [ ! "$save_dir" = "" ]; then
-    s3cmd sync --acl-public --disable-multipart $save_dir/* s3://$bucket_name/
+kbsign() {
+  cd $out_dir
```

Comment:
A `cd` here persists after this function exits -- could be confusing if calling code doesn't realize that the cwd has changed. Might be safer to put parens around these two lines, to contain the effect of the `cd`?